src/pages/GameNew.js

```diff
@@ -1,13 +1,162 @@
-import React from 'react'
+import React, { useState, currentUser } from "react";
+import { useNavigate } from "react-router-dom";
+import { Form, Row, Col, FormGroup, Label, Input, Button } from "reactstrap";
 
-const GameNew = () => {
+const GameNew = ({ createGame, currentUser }) => {
+  
+  const userPayload = currentUser?.split(".")[1];
+  const baseURL = userPayload.replace("-", "+").replace("_", "/");
+  const authUser = JSON.parse(atob(baseURL));
+
+  const [newGame, setNewGame] = useState({
+    title: "",
+    rating: "",
+    platform: "",
+    genre: "",
+    developer: "",
+    image: "",
+    summary: "",
+    release_date: "",
+    user_id: +authUser?.sub,
+  });
+
```

Comment:
If you are referencing the New Page from the instructor's Apartment App, the setup for the foreign key is based on that the backend and frontend are connected. The token is sent as the user authentication and has the user id embedded in its encryption. Since you are still using mock data, then you should be referencing the foreign key of your mock user. Please update to the current status of your project.